src/main/webapp/resources/js/pages/user/components/UserChangePasswordForm.jsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { Alert, Button, Form, Input, notification, Typography } from "antd";
+import { useEditUserDetailsMutation } from "../../../apis/users/users";
+
+/**
+ * React component to display the user change password form.
+ * @param {number} userId the identification number of the user
+ * @returns {*}
+ * @constructor
+ */
+export function UserChangePasswordForm({ userId }) {
+  const [editUser] = useEditUserDetailsMutation();
+  const [form] = Form.useForm();
+  const navigate = useNavigate();
+
+  const onFormFinish = (values) => {
+    editUser({ userId: userId, ...values })
+      .unwrap()
+      .then((payload) => {
+        notification.success({
+          message: i18n("UserChangePasswordForm.notification.success"),
+        });
+        form.resetFields();
+        navigate(`/${userId}/details`);
+      })
+      .catch((error) => {
+        const fields = Object.entries(error.data).map(([field, error]) => ({
+          name: field,
+          errors: [error],
+        }));
+        form.setFields(fields);
+      });
+  };
+
+  return (
+    <>
+      <Typography.Title level={5}>
+        {i18n("UserChangePasswordForm.title")}
+      </Typography.Title>
+      <Alert
+        message={i18n("UserChangePasswordForm.alert.title")}
+        description={
+          <Typography.Paragraph>
+            {i18n("UserChangePasswordForm.alert.description")}
+            <ul>
```

Comment:
Use the ant design List component here instead of using a <ul>

Reply:
Please see 12c424984c0c0da860baf19a23650aade436525a.